### ISSUE TITLE
Issue and PR templates

### ISSUE THREAD

**Issue by lapin-b** [go-gitea/gitea#3491]
- Gitea version (or commit ref): 1.4.0+rc1
- Git version: 2.16.1
- Operating system: Linux (Arch Linux)
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:
*Not relevant*

## Description
I have absolutely no idea how I can word this, but I'll do my best.

In a nutshell, if the template file exists in the repository (let's say `.gitea/ISSUE_TEMPLATE.md`), then Gitea can fetch the contents of the file and populate the textarea on the new issue template. Otherwise leave the textarea empty.

For example, GitHub implements this feature.

*PS: my wording can be terrible; English isn't my native language*

**Comment by kolaente** [comment]
This actually already exists:

Possible names for issues:
```
"ISSUE_TEMPLATE.md",
"issue_template.md",
".gitea/ISSUE_TEMPLATE.md",
".gitea/issue_template.md",
".github/ISSUE_TEMPLATE.md",
".github/issue_template.md",
```

And for PRs:
```
"PULL_REQUEST_TEMPLATE.md",
"pull_request_template.md",
".gitea/PULL_REQUEST_TEMPLATE.md",
".gitea/pull_request_template.md",
".github/PULL_REQUEST_TEMPLATE.md",
".github/pull_request_template.md",
```

We should add this somewhere to the docs.

**Comment by techknowlogick** [comment]
@kolaente I added a PR to add this information to the docs, please see: #3942
